Cargo.toml

```diff
@@ -59,6 +59,7 @@ lru = "0.12.5"
 mockall = "0.13.1"
 mpc-primitives = { path = "crates/primitives" }
 near-jsonrpc-client = "0.15.1"
+once_cell = "1.20.2"
```

Comment:
We shouldn't need OnceCell anymore since [LazyLock](https://doc.rust-lang.org/std/sync/struct.LazyLock.html) was stabilized.

Reply:
I see you used LazyCell instead as discussed in slack. Thanks!

Reply:
Please remove the `once_cell` workspace dependency here. I take it we don't have a check for unused workspace deps.